lib/components/dashboard/degree-info/distributionFunctions.tsx

```diff
@@ -66,9 +66,13 @@ export const getBoolExpr = (
     } else {
       concat = handleTagType(splitArr, index, course);
     }
+    if (index === 0 && concat === '&&!') {
+      boolExpr = boolExpr.concat('true');
```

Comment:
To clarify understanding, this means we will have "true&&!{next_expr}" for single prereq items?

Reply:
Yes!